cypress/integration/collections_spec.js

```diff
@@ -71,36 +78,35 @@ describe('Dashboard Collections Page', () => {
     it('collection page has button to edit the collection', () => {
       const name = 'MOD09GQ';
       const version = '006';
+
       cy.visit(`/#/collections/collection/${name}/${version}`);
       cy.contains('a', 'Edit').should('exist').as('editCollection');
       cy.get('@editCollection')
         .should('have.attr', 'href')
         .and('include', `#/collections/edit/${name}/${version}`);
       cy.get('@editCollection').click();
 
-      cy.contains('.heading--large', `Edit ${name}___${version}`);
-
-      // There are issues updating the react form which uses ace editor.
-      // Neither cy .clear nor ace editor works.
-      // As a workaround, we add additional parameter at the end of collection metadata
-      const meta = '"meta": "testmetadata"}';
-      cy.get('textarea').type(`{backspace},${meta}`, { force: true });
+      cy.contains('.heading--large', `Edit ${name}___${version}`).should('exist');
```

Comment:
You'll see a lot of changes like this. The reason why is that `cy.contains('.heading--large', `Edit ${name}___${version}`);` doesn't assert anything - it's just a selection method. So if we actually want to verify that these elements exist, we need to add `should` assertions

Reply:
If the content is not there, the `cy.contains` will fail due to timeout though.

Reply:
https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Default-Assertions

Reply:
From the documentation you linked:

> cy.contains() expects the element with content to eventually exist in the DOM.

So `cy.contains` has built-in retry behavior just like `cy.get` and therefore I wouldn't expect it to fail more or less often. However, I also just saw this in that documentation:

> All DOM based commands automatically wait for their elements to exist in the DOM.
> You never need to write .should('exist') after a DOM based command.

So I was wrong and you don't actually need the `should('exist')`. I'll remove it throughout this PR